poseidon2-starky/src/columns.rs

```diff
@@ -1,11 +1,33 @@
 /// The size of the state
 pub(crate) const STATE_SIZE: usize = 8;
+pub(crate) const SBOX_DEGREE: usize = 7;
+
+/// Poseidon constants
+pub(crate) const ROUNDS_F: usize = 4;
+pub(crate) const ROUNDS_P: usize = 22;
 
 /// The starting point of the column input
 pub(crate) const COL_INPUT_START: usize = 0;
 
+/// The starting point of the state after each 1st full round
+pub(crate) const COL_1ST_FULLROUND_STATE_START: usize = COL_INPUT_START + STATE_SIZE; // 8
+
+/// The starting point of the state[0] after each partial round
```

Comment:
it seems `state[0]` should be `state`. Or I misunderstood ?

Reply:
also feels like comment should say after each full round.

Reply:
In partial rounds, we only need to store state[0], which is the only value that receives a high degree update.

Reply:
updated the comments